dev/tools/pre-commit

```diff
@@ -56,19 +65,19 @@ then
 
```

Comment:
Add an ocamlformat call just above here

Reply:
Thanks, I should have realized. I was blinded by the fact that you initially told me that we would need two calls when in fact we needed three (if you count the `--check` in the test).